### PR TITLE
Git guide: some restructuring and slight improvments

### PR DESCRIPTION
**Pull request by maxteufel**

### CODE REVIEW

File: Git-guide,-TODO-'n'-notTODO.md [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
-* Don't make with vague suggestions. Always be specific of what you need.
-* We don't need any visionaries. Being productive in suggestions those someone could do is dangerous here.
-* When you make a big change, push it to your forked GitHub repo. and make a pull request.
-* Owners: Try not to push more than 3 commits straight to master at once. For small changes it should be rebased first and for big changes it should be put in a pull request.
\ No newline at end of file
+
+- Never force-push to a shared branch.
+- Don't constantly ask other people to update their content.
+- Use only constructive criticism, as being rude or acting annoyed could scare off a potential developer!
+- Don't beg anyone to teach you a something. Instead, ask them for guidance if you encounter a problem when following an online tutorial
```
Comment (tobbi) [inline Git-guide,-TODO-'n'-notTODO.md]: to teach you a something -> to teach you something.

File: Git-guide,-TODO-'n'-notTODO.md [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
+- Don't revert other people's work unless you're told to do so.
+- Don't push erroneous commits to master.
+- Don't make with vague suggestions. Always be specific of what you need.
+- We don't need any visionaries. Being productive in suggestions those someone could do is dangerous here.
```
Comment (tobbi) [inline Git-guide,-TODO-'n'-notTODO.md]: Strange sentence I don't understand.

File: Git-guide,-TODO-'n'-notTODO.md [hunk 3]
```diff
@@ -1,42 +1,45 @@
...
+- When you make a big change, push it to your forked GitHub repo. and make a pull request.
```
Comment (tobbi) [inline Git-guide,-TODO-'n'-notTODO.md]: Remove dot before "and"

File: Git-guide,-TODO-'n'-notTODO.md [hunk 4]
```diff
@@ -1,42 +1,45 @@
...
+
+- Never force-push to a shared branch
+- Don't constantly ask other people to update their content
+- Use only constructive criticism, as being rude or acting annoyed could scare off a potential developer!
+- Don't beg anyone to teach you something. Instead, ask them for guidance if you encounter a problem when following an online tutorial
+- Don't revert other people's work unless you're told to do so
+- Don't push erroneous commits to master
+- Don't be too vague, try to be as specific as possible
+- We don't need any visionaries. Being productive in suggestions those someone could do is dangerous here
+- When you make a big change, push it to your forked GitHub repo. and make a pull request
```
Comment (tobbi) [inline Git-guide,-TODO-'n'-notTODO.md]: There's a dot after 'repo'.

File: Git-guide,-TODO-'n'-notTODO.md [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
\ No newline at end of file
+
+- Never force-push to a shared branch
+- Don't constantly ask other people to update their content
+- Use only constructive criticism, as being rude or acting annoyed could scare off a potential developer!
+- Don't beg anyone to teach you something. Instead, ask them for guidance if you encounter a problem when following an online tutorial
+- Don't revert other people's work unless you're told to do so
+- Don't push erroneous commits to master
+- Don't be too vague, try to be as specific as possible
+- We don't need any visionaries. Being productive in suggestions those someone could do is dangerous here
```
Comment (tobbi) [inline Git-guide,-TODO-'n'-notTODO.md]: "Being productive in suggestions those someone could do is dangerous here" <-- I still don't understand that sentence.

Comment (maxteufel) [reply Git-guide,-TODO-'n'-notTODO.md]: I'll remove that entirely.

**Pull request merged**